### ISSUE TITLE
Spectral similarity in DIA-NN

### ISSUE THREAD

**Issue by candidaorelmex** [vdemichev/DiaNN#923]
Hi,

For one of my analyses I recieved a spectral similarity distribution that resembles a normal distribuiton with the mean around 0.4 and a standard deviation of 0.18. When I compare this to spectral similarities/angles that I recieve from MSFragger, the DIA-NN spectral similarity values seem very low (~0.9 vs 0.4).

1) What spectral similarity value  for a PSM would you expect for a true-positive precursor identification? 
2) What metric is used to calculate spectral similarity?

Thanks in advance!

**Comment by vdemichev** [comment]
'Spectral similarity' in DIA-NN is an experimental (= no guarantees here) proxy for the quality of theoretical vs observed fragment intesnities match. Proxy in the sense that higher values mean a better match, i.e. the absolute values are not supposed to be interpreted directly. The reason for introducing such a metric is because a regular cosine similarity between spectra is a very poor measure for DIA data.

Whether a peptide is true/false positive is decided solely on its q-value/PEP - all other metrics is just auxillary information which can potentially be used for other purposes (i.e. site localisation in phospho) - but in this case it has not been tested properly, hence experimental status.

**Comment by candidaorelmex** [comment]
I assume absolute spectral angle values are not as good a metric for DIA-based PSMs that DDA ones because DIA MS2 spectra are much more crowded?

Regarding the the q-value/PEP: this requres the comparison of target PSMs and decoy PSMs -> where do decoys come from when doing a library search? Do you have a recommendation for a paper that explains how target-decoy strategies work for DIA library searches?

Many thanks!

**Comment by vdemichev** [comment]
Yes, a single highly intense interference peak affects the spectral angle a lot.

**Comment by vdemichev** [comment]
Decoys are generated by DIA-NN on the fly, using the algorithm described in https://www.nature.com/articles/s41592-019-0638-x
